resources/build/build-utils.sh

```diff
@@ -843,10 +871,12 @@ builder_report() {
   local scope="[$THIS_SCRIPT_IDENTIFIER] "
 
   if _builder_item_in_array "$action$target" "${_builder_current_actions[@]}"; then
-    if [ $result == success ]; then
+    if [[ $result == success ]]; then
       echo "${COLOR_GREEN}## $scope$action$target completed successfully${COLOR_RESET}"
+    elif [[ $result == failure ]]; then
+      echo "${COLOR_RED}## $scope$action$target failed${COLOR_RESET}"
```

Comment:
Cleanup, allowing us to do `builder_finish_action "something went booboo" action:target` and give additional detail...

Reply:
The whole `... failed.  Result: failure` bit was kind of redundant too - nice to see the option to be less redundant in the general case and to allow a better-quality report in the other case.